### PR TITLE
Fixed issue with setting correct status bar color in iOS

### PR DESCRIPTION
**Pull request by yakivmospan**

### CODE REVIEW

File: lib/main.dart [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import 'package:flutter/material.dart';
+import 'package:flutterapp/resources/app_colors.dart';
```
Comment (ababicheva) [inline lib/main.dart]: but we don't use app_colors in thi file

File: lib/presentation/home_route.dart [hunk 2]
```diff
@@ -116,18 +114,18 @@ class _BottomTabsView extends StatelessWidget {
 }
 
 Color _getStatusBarColor(int pageIndex) {
-  if(pageIndex == profileTabIndex) {
+  if (pageIndex == profileTabIndex) {
     return AppColors.dark;
   } else {
-    return AppColors.white;
+    return AppColors.paleGray;
```
Comment (ababicheva) [inline lib/presentation/home_route.dart]: why it's paleGray? we have white status bar color also

File: lib/main.dart (hunk 1 above)
Comment (yakivmospan) [reply lib/main.dart]: Thanks, leftover from debug code

File: lib/presentation/home_route.dart (hunk 2 above)
Comment (yakivmospan) [reply lib/presentation/home_route.dart]: Mostly for Order screen, will update for all tabs we got, thanks

**Review by ababicheva — APPROVED** [review]

**Review by vyatsykiv — APPROVED** [review]

**Pull request merged**